ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -127,11 +114,11 @@ public protocol DrawerControllerDelegate: AnyObject {
 public typealias MSDrawerController = DrawerController
 
 @objc(MSFDrawerController)
-open class DrawerController: UIViewController {
+open class DrawerController: UIViewController, FluentUIWindowProvider {
```

Comment:
Can you rename MSFDrawerControllerLegacyDelegate back to MSFDrawerControllerDelegate